frontend/src/lib/constants.tsx

```diff
@@ -208,35 +208,35 @@ export const FEATURE_FLAGS = {
-    [AvailableFeature.DASHBOARD_COLLABORATION]: LicensePlan.Scale,
-    [AvailableFeature.INGESTION_TAXONOMY]: LicensePlan.Scale,
-    [AvailableFeature.PATHS_ADVANCED]: LicensePlan.Scale,
-    [AvailableFeature.CORRELATION_ANALYSIS]: LicensePlan.Scale,
-    [AvailableFeature.GROUP_ANALYTICS]: LicensePlan.Scale,
-    [AvailableFeature.MULTIVARIATE_FLAGS]: LicensePlan.Scale,
-    [AvailableFeature.EXPERIMENTATION]: LicensePlan.Scale,
-    [AvailableFeature.TAGGING]: LicensePlan.Scale,
-    [AvailableFeature.BEHAVIORAL_COHORT_FILTERING]: LicensePlan.Scale,
-    [AvailableFeature.WHITE_LABELLING]: LicensePlan.Scale,
-    [AvailableFeature.DASHBOARD_PERMISSIONING]: LicensePlan.Enterprise,
-    [AvailableFeature.PROJECT_BASED_PERMISSIONING]: LicensePlan.Enterprise,
-    [AvailableFeature.SAML]: LicensePlan.Enterprise,
-    [AvailableFeature.SSO_ENFORCEMENT]: LicensePlan.Enterprise,
-    [AvailableFeature.SUBSCRIPTIONS]: LicensePlan.Scale,
-    [AvailableFeature.APP_METRICS]: LicensePlan.Scale,
-    [AvailableFeature.RECORDINGS_PLAYLISTS]: LicensePlan.Scale,
-    [AvailableFeature.ROLE_BASED_ACCESS]: LicensePlan.Enterprise,
-    [AvailableFeature.RECORDINGS_FILE_EXPORT]: LicensePlan.Scale,
-    [AvailableFeature.RECORDINGS_PERFORMANCE]: LicensePlan.Scale,
-    [AvailableFeature.SURVEYS_STYLING]: LicensePlan.Scale,
-    [AvailableFeature.SURVEYS_MULTIPLE_QUESTIONS]: LicensePlan.Scale,
-    [AvailableFeature.SURVEYS_TEXT_HTML]: LicensePlan.Scale,
-    [AvailableFeature.DATA_PIPELINES]: LicensePlan.Scale,
+    [AvailableFeature.ZAPIER]: 'scale',
+    [AvailableFeature.ORGANIZATIONS_PROJECTS]: 'scale',
+    [AvailableFeature.SOCIAL_SSO]: 'scale',
+    [AvailableFeature.DASHBOARD_COLLABORATION]: 'scale',
+    [AvailableFeature.INGESTION_TAXONOMY]: 'scale',
+    [AvailableFeature.PATHS_ADVANCED]: 'scale',
+    [AvailableFeature.CORRELATION_ANALYSIS]: 'scale',
+    [AvailableFeature.GROUP_ANALYTICS]: 'scale',
+    [AvailableFeature.MULTIVARIATE_FLAGS]: 'scale',
+    [AvailableFeature.EXPERIMENTATION]: 'scale',
+    [AvailableFeature.TAGGING]: 'scale',
+    [AvailableFeature.BEHAVIORAL_COHORT_FILTERING]: 'scale',
+    [AvailableFeature.WHITE_LABELLING]: 'scale',
+    [AvailableFeature.DASHBOARD_PERMISSIONING]: 'enterprise',
+    [AvailableFeature.PROJECT_BASED_PERMISSIONING]: 'enterprise',
+    [AvailableFeature.SAML]: 'enterprise',
+    [AvailableFeature.SSO_ENFORCEMENT]: 'enterprise',
+    [AvailableFeature.SUBSCRIPTIONS]: 'scale',
+    [AvailableFeature.APP_METRICS]: 'scale',
+    [AvailableFeature.RECORDINGS_PLAYLISTS]: 'scale',
+    [AvailableFeature.ROLE_BASED_ACCESS]: 'enterprise',
+    [AvailableFeature.RECORDINGS_FILE_EXPORT]: 'scale',
+    [AvailableFeature.RECORDINGS_PERFORMANCE]: 'scale',
+    [AvailableFeature.SURVEYS_STYLING]: 'scale',
+    [AvailableFeature.SURVEYS_MULTIPLE_QUESTIONS]: 'scale',
+    [AvailableFeature.SURVEYS_TEXT_HTML]: 'scale',
+    [AvailableFeature.DATA_PIPELINES]: 'scale',
```

Comment:
@xrdt  Why this? I think this is worse haha

Reply:
Hmm, I think can simply remove this field from the billing json fixture and revert it.